src/common/media.cpp

```diff
@@ -76,12 +77,134 @@ char getByte(GCodeFilter::State *state);
 static char gcode_buffer[MAX_CMD_SIZE + 1]; // + 1 for NULL char
 static GCodeFilter gcode_filter(&getByte, gcode_buffer, sizeof(gcode_buffer));
 static uint32_t media_loop_read = 0;
-static const constexpr uint32_t MEDIA_LOOP_MAX_READ = 4096;
 
 media_state_t media_get_state(void) {
     return media_state;
 }
 
+#define PREFETCH_SIGNAL_START 1
+#define PREFETCH_SIGNAL_STOP  2
+#define PREFETCH_SIGNAL_FETCH 4
+// These buffers are HUGE. We need to rework the prefetcher logic
+// to be more efficient and add compression.
+#define FILE_BUFF_SIZE 5120
+static char prefetch_buff[2][FILE_BUFF_SIZE];
```

Comment:
Are we sure that these buffers + functions are not used from anywhere else? E.g. PrusaLink trying to fetch thumbnails?

Reply:
I don't understand.